src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,21 @@
+import "./avatar.scss";
+
+import React, { FC } from "react";
+
+import avatar from "./media/user.svg";
+
+interface Props {
+	height: number;
+	width: number;
+	url?: string;
+}
+
+const Avatar: FC<Props> = ({ url = avatar, height, width }) => {
```

Comment:
### W reactcie tak nie ustawisz domyślnej wartości dla propsa.

Możesz zrobić to przez atrybut defaultProps  
https://medium.com/@martin_hotell/react-typescript-and-defaultprops-dilemma-ca7f81c661c7

Albo prostszy sposób, w kodzie szablonu: 
![image](https://user-images.githubusercontent.com/39082174/113393168-9048ce00-9396-11eb-96e1-0b926833a4c6.png)